compiler/include/driver.h

```diff
@@ -89,6 +89,8 @@ extern const char* CHPL_TARGET_PLATFORM;
 extern const char* CHPL_TARGET_COMPILER;
 extern const char* CHPL_ORIG_TARGET_COMPILER;
 extern const char* CHPL_TARGET_ARCH;
+extern const char* CHPL_RUNTIME_ARCH;
+extern const char* CHPL_TARGET_ARCH_ARG;
```

Comment:
This isn't a particularly meaningful name to me. I don't have any great suggestions though. Maybe `CHPL_TARGET_MAPPED_ARCH`?

Also maybe swap the ordering so it's next to `CHPL_TARGET_ARCH`?

Reply:
I'm fine with CHPL_TARGET_MAPPED_ARCH if that's more satisfying to you. It doesn't say what it's mapped to, though...

Reply:
I don't know if I like it any better, but it was the only suggestion I could think of

Reply:
It is mapped to a `cpu-type` according to `gcc` man pages. Would `CHPL_TARGET_CPUTYPE` or something along those lines be any less ambiguous?

Reply:
I like combining the proposals for this weird variable, so we get `CHPL_TARGET_ARCH_BACKEND_ARG`